dts/x86/quark_x1000.dtsi

```diff
@@ -59,5 +59,17 @@
 
 			status = "disabled";
 		};
+
+		i2c0: i2c@0x90007000 {
+			compatible = "snps,designware-i2c";
+			#address-cells = <1>;
+			#size-cells = <0>;
+			reg = <0x90007000 0x400>;
+			interrupts = <18 IRQ_TYPE_LEVEL_LOW>;
```

Comment:
Is IRQ_TYPE_LEVEL_LOW the IRQ priority?

Reply:
No that's the interrupt flags. I don't know if it's specific to ioapic.